Tests/DatadogTests/Datadog/RUM/RUMMonitor/Scopes/RUMViewScopeTests.swift

```diff
@@ -1139,4 +1139,38 @@ class RUMViewScopeTests: XCTestCase {
         XCTAssertEqual(event.model.view.action.count, 0, "All actions, including ApplicationStart action should be dropped")
         XCTAssertEqual(event.model.dd.documentVersion, 1, "It should record only one view update")
     }
+
+    // MARK: Integration with Crash Context
+
+    func testWhenViewIsStarted_thenItUpdatesLastRUMViewEventInCrashContext() throws {
```

Comment:
I added this missing test, because now `RUMWithCrashContextIntegration` is injected through DI and we're able to configure the mock.